src/QMCWaveFunctions/Jastrow/PadeFunctors.h

```diff
@@ -240,10 +240,18 @@ struct PadeFunctor : public OptimizableFunctorBase
       if (ia > -1)
       {
         int i = 0;
-        if (Opt_A)
-          A = myVars[i++] = active[ia++];
-        if (Opt_B)
-          B0 = myVars[i] = active[ia];
+        if (Opt_A) {
+          //A = myVars[i++] = active[ia++];
+          myVars[i]=active[ia];
+          A = std::real( myVars[i] );
+          i++;
+          ia++;
+        }
```

Comment:
You can simply write
```
A = std::real(myVars[i++] = active[ia++]);
```
Please clean up all similar places.

Reply:
Fixed.